pkg/util/resources.go

```diff
@@ -40,6 +40,21 @@ func ParseAPIGroupsString(apiGroups string) sets.Set[string] {
 	return groupsSet
 }
 
+func ParseControllersString(controllers string) sets.Set[string] {
+	if controllers == "*" {
+		return nil
+	}
+
+	return sets.New(allToLowerCase(strings.Split(controllers, ","))...)
```

Comment:
I think that allowing variation in case is more complexity than we need here.

Reply:
Just want to clarify: Is the suggestion to (a) remove the case check/conversion, or (b) reject inputs with upper cases?

Reply:
The suggestion is the just use one case consistently. All the provision and consumption of these strings is in the same repo. We have no user community with an established practice of using both cases. Just pick one and go with it.

Reply:
OK, rejected inputs with upper case letters.

Reply:
I am trying to suggest something even simpler. There is a valid set of controller names. Those are literal exact strings, end of story. Any invalid controller name is either simply ineffective or is reported as an error. The user is responsible for passing valid controller names.